adoc/retail_chap_formulas.adoc

```diff
@@ -111,69 +111,35 @@ Use only alphanumeric characters for the branch identifier.
-* In the [guimenu]``Name`` field, use the reverse zone for the configured IP range (for example: [systemitem]``1.168.192.in-addr.arpa``).
-* In the [guimenu]``Type`` field, select [systemitem]``master``
-. In the [guimenu]``Available Zones`` section, use these parameters for Zone1:
-* In the [guimenu]``Name`` field, use the FQDN of the  branch server (for example: [systemitem]``branch1.mycompany.org``).
-* In the [guimenu]``File`` field, type the name of your configuration file *TODO: Check this* (for example: [systemitem]``branch1.txt``).
-. In the [guimenu]``Start of Authority (SOA)`` section, use these parameters for Zone1:
-* In the [guimenu]``Nameserver (Ns)`` field, use the FQDN of the  branch server (for example: [systemitem]``branch1.mycompany.org``).
-* In the [guimenu]``Contact`` field, use the email address for the domain administrator.
-* Leave all other fields as their default values.
-. In the [guimenu]``Records`` section, in subsection [guimenu]``A``, click on [btn]``Add Item`` and use these parameters to set up an A record for Zone1:
-* In the [guimenu]``Hostname`` field, use the hostname of the branch server (for example: [systemitem]``branchserver``).
-* In the [guimenu]``IP`` field, use the IP address of the branch server (for example, [systemitem]``192.168.1.1``).
-. In the [guimenu]``Records`` section, subsection [guimenu]``NS``, click on [btn]``Add Item`` and use these parameters to set up an NS record for Zone1:
-* In the input box, use the hostname of the branch server (for example: [systemitem]``branchserver``).
-. In the [guimenu]``Records`` section, subsection [guimenu]``CNAME``, click on [btn]``Add Item`` and add the hostname of the branch server in each of these fields:
-* [systemitem]``tftp``
-* [systemitem]``ftp``
-* [systemitem]``dns``
-* [systemitem]``dhcp``
-* [systemitem]``salt``
-. Set up Zone2 using the same parameters as for Zone1, but ensure you use the reverse  details.
-. Click [btn]``Save Formula`` to save your configuration
-. Apply the highstate.
+// TBD
 
 
 
-////
-Moved from Admin chapter. Checking which procedure is correct. LKB
+[[retail.sect.formulas.bind]]
+== Bind Formula
 
-=== Configure the Saltboot formula
+The Bind formula is used to configure the boot image on a terminal.
 
-.Procedure: Configuring the Saltboot formula
 
-. Open the details page for your new hardware group, and navigate to the [guimenu]``Formulas`` tab.
-. Select the [systemitem]``saltboot-formula`` and click btn:[Save] to save your changes.
-. An additional [guimenu]``saltboot`` tab will become available from the top menu, click on it to configure the new formula.
-+
 [NOTE]
 ====
 There are many different configurations possible.
 This procedure outlines a simple, single-disk, three-partition configuration.
 Adjust it to suit your own environment.
 ====
+
+
+.Procedure: Configuring bind
+
+. Open the details page for your new hardware group, and navigate to the [guimenu]``Formulas`` tab.
```

Comment:
This is a saltboot configuration. Please move it up where the saltboot formual is talked about